src/services/Auth/login.js

```diff
@@ -3,38 +3,36 @@ import { useNavigate } from "react-router-dom";
 import { useProducts } from "../../context/products";
 import { useToast } from "../../context/toast";
 import { useAxios } from "../../hooks";
-import { MESSAGES } from "../../utils/constants";
+import { MESSAGES } from "../../utils";
 
```

Comment:
Is there any specific reason of importing `MESSAGE` in caps?

Reply:
Its a constant. Constants are usually in caps

Reply:
> Its a constant. Constants are usually in caps

got it!